index.js

```diff
@@ -161,12 +160,36 @@ downloadButton.on("click", function () {
     link.click();
     downloadButton.text("Downloaded!");
   } catch {
-    console.log(
+    log(
       'Image can\'t be downloaded unless running on a server. Try "python -m SimpleHTTPServer 8000"'
     );
   }
 });
 
+//Create logging functions to toggle logging between dev and prod modes
+function log(text, style) {
+  if (devMode) {
+    if (style) {
+      console.log(text, style);
+    }
```

Comment:
Can we do something like `function log(text, style="")`. This will set the default style to nothing so we don't need the inner conditionals.